packages/bpk-component-popover/readme.md

```diff
@@ -42,23 +42,30 @@ class App extends Component {
 
   render() {
     return (
-      <BpkPopover
-        id="my-popover"
-        target={
-          <BpkButton onClick={this.openPopover}>Open</BpkButton>
-        }
-        onClose={this.closePopover}
-        isOpen={this.state.isOpen}
-        label="My popover"
-        closeButtonText="Close"
-      >
-        <BpkText>My popover content</BpkText>
-      </BpkPopover>
+      <div>
+        <BpkPopover
+          id="my-popover"
+          target={
+            <BpkButton onClick={this.openPopover}>Open</BpkButton>
+          }
+          onClose={this.closePopover}
+          isOpen={this.state.isOpen}
+          label="My popover"
+          closeButtonText="Close"
+          renderTarget={this.renderTarget}
+        >
+          <BpkText>My popover content</BpkText>
+        </BpkPopover>
+        <div ref={(target) => { this.renderTarget = target; }} />
+      </div>
     );
   }
 }
 ```
 
+**Theming:** In order to theme the modal a `renderTarget` needs to be supplied as a DOM reference of a node in
```

Comment:
I have this wording and api changed locally - will submit the PR shortly.

Reply:
Cool, I'll update once you submit so we don't need a new PR